ibis/backends/base/sql/registry/literal.py

```diff
@@ -21,7 +21,13 @@ def _boolean_literal_format(translator, op):
 
 
 def _string_literal_format(translator, op):
```

Comment:
Note: this is in the base sql class, not BigQuery-specific. I don't know if the other backends suffer the same limitation of no unescaped newlines in string literals, but it seems as though it wouldn't do any harm to have it.